### ISSUE TITLE
Xinput Mischief Data

### ISSUE THREAD

**Issue by shinra358** [Ryochan7/DS4Windows#38]
<a href='https://postimg.org/image/nnhn1ca4d/' target='_blank'><img src='https://s6.postimg.org/nnhn1ca4d/ds4444.png' border='0' alt='postimage'/></a>

**Comment by shinra358** [comment]
Even though things are still working good this time around, I'm still getting these in the logs. What do they mean?

**Comment by Ryochan7** [comment]
Some of those errors would probably be from a normal disconnect. I have decided to stop showing those messages since it would be difficult to distinguish a USB unplug event from an actual device error.

Also, I just happened to stumble upon an error regarding an XInput device plugin routine caused by a bad copy paste job. The wrong index value was being used when running the hotplug routine which could cause problems when using multiple controllers.

https://drive.google.com/open?id=0B6yGHDx0CFzDeXlVZmVMSUxOOWM
